jctools-core/src/main/java/org/jctools/queues/atomic/MpmcAtomicArrayQueue.java

```diff
@@ -92,31 +133,34 @@ public boolean offer(final E e) {
             throw new NullPointerException();
         }
         final int mask = this.mask;
-        final int capacity = mask + 1;
+        final long capacity = mask + 1;
         final AtomicLongArray sBuffer = sequenceBuffer;
-
         long pIndex;
         int seqOffset;
         long seq;
-        long cIndex = Long.MIN_VALUE;// start with bogus value, hope we don't need it
+        // start with bogus value, hope we don't need it
+        long cIndex = Long.MIN_VALUE;
         do {
             pIndex = lvProducerIndex();
             seqOffset = calcSequenceOffset(pIndex, mask);
             seq = lvSequence(sBuffer, seqOffset);
-            if (seq < pIndex) { // consumer has not moved this seq forward, it's as last producer left
```

Comment:
This appears to be a bug in JavaParser caused when there's a comment on the following line to an if statement with a comment